setup.py

```diff
@@ -22,6 +22,7 @@
 }
 uwsgi_max_extras = {"uwsgi==2.0.*"}
 uwsgi_min_extras = {"uwsgi==2.0.14"}
+gunicorn_min_extras = {"gunicorn==0.16.1"}
```

Comment:
you'll now neeed to change `gunicorn_extras` to be `gunicorn_max_extras` just like uwsgi